UPGRADE.md

```diff
@@ -1,3 +1,7 @@
+# Upgrading from v2.1.1 to v2.2.0
```

Comment:
This doesn't seem right. I understand that you were misled by the other entry (which is also wrong), but specifying the patch version here doesn't make sense.
Please change it to "from 2.1 to 2.2". Please change the other one accordingly ("from 2.0 to 2.1").